src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedTryDepthCheck.java

```diff
@@ -48,7 +48,27 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * try { // ...
+ *     try { // ...
+ *         try { // violation
+ *         } catch (Exception e) { // violation
+ *         }
+ *     } catch (Exception e) { // on Exception
+ *     }
+ * } catch (Exception e) { // on Exception
+ * }
+ *</pre>
+ *<p>Example of correct code:</p>
+ *<pre>
+ * try { // ...
+ *     try { // ...  OK for default setup
```

Comment:
Please move code example after config example.

Reply:
I have moved default code example after config example. please tell me if there is wrong formatting.

Reply:
It is incorrect right now. 
It should be like 
<default config example>
<code with ok and violation for default>

<not default config example (this case with "max" property)>
<code with ok and violation for default>